distributions/openhab-online/readme.md

```diff
@@ -0,0 +1,37 @@
+openHAB online distro
+---------------------
+
+
+The online distro doesn't contain many features.
+You'll need to install those manually for now.
+
+
+1. start openhab
+	start.sh
+
+2. view available features
+	feature:list
+
+3. install a feature e.g. install rest docs
+	feature:install openhab-misc-restdocs
```

Comment:
> It means that the minimal runtime already requires an http server, which is actually pretty big. If you think of an "headless" openHAB instance, e.g. for only running a specific binding and sending data through MQTT, an http server is not needed.

OK, we have a difference in understanding here. I always considered minimal runtime as the smallest possible openHAB 'master' instance. For running only a specific binding in a slave configuration, I believe a dedicated 'slave/node' runtime would be more appropriate as we could tailor it for that use, .e.g for me a slave runtime, would:
- be preconfigured to run in a slave configuration
- be autodiscovered by the master instance
- be fully configurable from the master instance


> why purely relying on an admin UI is not good

I agree that config should also be possible. But UI is in my opinion also needed for the master instance. If a user finds it too difficult to enter 'feature:install openhab-demo' on the command line, then filling out a config file will be a challenge too :wink: 

> that it can be backed up or automatically constructed

Backed up, I understand. I'm not sure what you mean with automatically constructed.